lib/jobs/run-work-items.js

```diff
@@ -207,6 +208,37 @@ function runWorkItemsJobFactory(
             );
         });
     };
+    
+    /**
+     * @function _executeRedfishPoller
+     * @description execute the redfish worker item 
+     */
+    RunWorkItemsJob.prototype._executeRedfishPoller = function (workItem) {
```

Comment:
This copies the same pattern as the other poller work item code.  Are you suggesting those should be refactored?